src/main/cpp/include/api/genomicsdb.h

```diff
@@ -323,7 +318,7 @@ class GenomicsDB {
   /**
    * Constructor to the GenomicsDB Query API with configuration json files
    *   query_configuration - describe the query configuration in either a JSON file or string
-   *   query_configuration_type - type of query configuration, could be a JSON_FILE or JSON_STRING
+   *   query_configuration_type - type of query confi1guration, could be a JSON_FILE or JSON_STRING
```

Comment:
Minor typo, extra `1` character in `configuration`.

Reply:
Oops!